tox.ini

```diff
@@ -157,6 +157,7 @@ deps =
     aiobotocore04: aiobotocore>=0.4,<0.5
     aiobotocore03: aiobotocore>=0.3,<0.4
     aiobotocore02: aiobotocore>=0.2,<0.3
+    aiobotocore02: multidict==4.5.2
```

Comment:
```suggestion
    aiobotocore02: multidict!=4.6.0
```

Reply:
Just tested it with the release of 4.6.1 from hours ago and it's still failing. If time permits, I'll try to look into the changes https://github.com/aio-libs/multidict/compare/v4.5.2...v4.6.0

Reply:
Do you have what the failure is? Probably something we should make sure we support.

Reply:
I put the log in a gist https://gist.github.com/codeboten/6aeb044cd1578790bfbc128378eb2d79

Reply:
Yeah that looks like a bug in our (test) code. We need to fix it.

Reply:
I actually misread the data. This is triggered by a test but it's not our code, it's just that this old version of aiobotocore cannot work with the new multidict release.
It means that nobody can install this version of aiobotocore without some requirements hack.

Reply:
Why not go with `multidict<=4.6.0` in this case?